src/wire/libevent_server.cpp

```diff
@@ -62,9 +68,19 @@ void Signal_Callback(UNUSED_ATTRIBUTE evutil_socket_t fd,
   event_base_loopexit(base, NULL);
 }
 
+void Status_Callback(UNUSED_ATTRIBUTE evutil_socket_t fd,
+                     UNUSED_ATTRIBUTE short what, void *arg) {
```

Comment:
Will `fd` and `server` be used in the future? If yes, you may want to add a `TODO` here. If not, you may need to update the interface of this function.